backend/clubs/permissions.py

```diff
@@ -263,9 +297,8 @@ def has_object_permission(self, request, view, obj):
                 return True
 
             # otherwise, allow club officers to edit and delete comments
-            membership = Membership.objects.filter(
-                person=request.user, club__code=view.kwargs["club_code"]
-            ).first()
+            object = Club.objects.get(code=view.kwargs["club_code"])
```

Comment:
Instead of doing this, it might be more efficient to use the `obj` object, which is a `QuestionAnswer` model.